addons/sourcemod/translations/zombieriot.phrases.zombienames.txt

```diff
@@ -4,647 +4,777 @@
+		"pt"		"Xeno Spy Facestabber"
 	}
 	"Xeno Soldier Rocketeer"
 	{
 		"en" 		"Xeno Soldier Rocketeer"
 		"ru" 		"Ксеносолдат-ракетчик"
+		"pt"		"Xeno Soldado Bombardeiro"
 	}
 	"Xeno Soldier Minion"
 	{
 		"en" 		"Xeno Soldier Minion"
 		"ru" 		"Ксеносолдат-приспешник"
+		"pt"		"Xeno Soldado Lacaio"
 	}
 	"Xeno Soldier Giant Summoner"
 	{
 		"en" 		"Xeno Soldier Giant Summoner"
 		"ru" 		"Гигантский ксеносолдат-призыватель"
+		"pt"		"Xeno Soldado Gigante Invocador"
 	}
 	"Xeno Spy Thief"
 	{
 		"en" 		"Xeno Spy Thief"
 		"ru" 		"Вор-ксеношпион"
+		"pt"		"Xeno Spy Ladrão"
 	}
 	"Xeno Spy Trickstabber"
 	{
 		"en" 		"Xeno Spy Trickstabber"
 		"ru" 		"Ксеношпион-трюкач"
+		"pt"		"Xeno Spy Trickstabber"
 	}
 	"Xeno Half Cloaked Spy"
 	{
 		"en" 		"Xeno Half Cloaked Spy"
 		"ru" 		"Полуневидимый ксеношпион"
+		"pt"		"Xeno Spy Parcialmente Invisivel"
 	}
 	"Xeno Sniper Main"
 	{
 		"en" 		"Xeno Sniper Main"
 		"ru" 		"Ксеноснайпер-классасин"
+		"pt"		"Xeno Sniper Main"
 	}
 	"Xeno Demoknight Main"
 	{
 		"en" 		"Xeno Demoknight Main"
 		"ru" 		"Рыцарь-ксеноподрывник-классасин"
+		"pt"		"Xeno Demoknight Main"
 	}
 	"Xeno Battle Medic Main"
 	{
 		"en" 		"Xeno Battle Medic Main"
 		"ru" 		"Боевой ксеномедик-классасин"
-	}	
+		"pt"		"Xeno Medic Main de Combate"
+	}
 	"Xeno Giant Pyro Main"
 	{
 		"en" 		"Xeno Giant Pyro Main"
 		"ru" 		"Гигантский ксеноподжигатель-классасин"
+		"pt"		"Xeno Pyro Main Gigante"
 	}
 	"Xeno Combine Deutsch Ritter"
 	{
 		"en" 		"Xeno Combine Deutsch Ritter"
 		"ru" 		"Немецкий ксенокомбайн-рыцарь"
+		"pt"		"Xeno Deutsch Ritter da Combine"
 	}
 	"Xeno X10 Spy Main"
 	{
 		"en" 		"Xeno X10 Spy Main"
 		"ru" 		"Ксеношпион-классасин сервера X10"
+		"pt"		"Xeno Spy Main do X10"
 	}
 
 	"Nazi Panzer"
 	{
 		"en" 		"Nazi Panzer"
 		"ru" 		"Нацистский панцер"
+		"pt"		"Tanque Nazista"
 	}
 	"Bob the Overgod of gods and destroyer of multiverses"
 	{
 		"en" 		"Bob the Overgod of gods and destroyer of multiverses"
 		"ru" 		"Боб верховный бог богов и разрушитель мультивселенных"
+		"pt"		"Bob o Poderoso e destruir de multiversos"
 	}
 	"Revived Combine DDT"
 	{
 		"en" 		"Revived Combine DDT"
 		"ru" 		"Оживлённый комбайн DDT"
+		"pt"		"Combine DDT Revivido"
 	}
 	"Spookmaster Boner"
 	{
 		"en" 		"Spookmaster Boner"
 		"ru" 		"Костяной мастер-призрак"
+		"pt"		"Esqueleto Mestre dos Sustos"
 	}
 	"Cured Father Grigori"
 	{
 		"en" 		"Cured Father Grigori"
 		"ru" 		"Исцелённый отец Григорий"
+		"pt"		"Senhor Grigori Curado"
 	}
 	"Combine Mage"
 	{
 		"en" 		"Combine Mage"
 		"ru" 		"Комбайн маг"
+		"pt"		"Mago da Combine"
 	}
 	"Bloon"
 	{
 		"en" 		"Bloon"
 		"ru" 		"Шар"
+		"pt"		"Balão"
 	}
 	"Massive Ornery Air Blimp"
 	{
 		"en" 		"Massive Ornery Air Blimp"
 		"ru" 		"Массивный злобный дирижабль"
+		"pt"		"Dirigível Massivamente Grande"
 	}
 	"Brutal Floating Behemoth"
 	{
 		"en" 		"Brutal Floating Behemoth"
 		"ru" 		"Жесточайший летающий бегемот"
+		"pt"		"Brutamontes Flutuante" // i am confused on what this is
```

Comment:
what in the hell is this enemy supposed to be?

Reply:
bloons tower defense things